pages/models/generations/Sources/VSC/PhasorGridFollowingVSC/index.md

```diff
@@ -0,0 +1,325 @@
+---
+layout: page 
+title: Full-Phasor Grid Following Voltage Source Converter 
+tags: [Opensource, Phasor, voltage source, converter, wind, pv, hdvc] 
+date: 26/06/2024 
+last-updated: 09/07/2024
+id: #165
+authors: Carlos Alegre (eRoots)
+reviewers: Eduardo Prieto Araujo (UPC), Josep Fanals Batllori (eRoots)
+---
+
+
+## Context
+
+Voltage Source Converters (VSC) are widely used in power systems for a variety of applications, such as wind and photovoltaic generation, High Voltage Direct Current (HVDC) transmission, and Flexible AC Transmission Systems (FACTS). The model described here is the Full-Phasor model of a grid-following VSC, which is a type of VSC that is synchronized with the grid, using its frequency and phase. This model, obtained from the many works developed at CITCEA-UPC (Centre d'Innovació Tecnològica en Convertidors Estàtics i Accionaments) such as [[1]](#1), [[2]](#2) and [[3]](#3), is useful for studying medium to low-frequency phenomena of the VSC and its interaction with the grid.
+    
+## Model use, assumptions, validity domain and limitations
+
+The model described allows performing phasor studies of the dynamics of a grid-following voltage source converter. It is specially useful in applications where there are slow-transients that want to be studied, such as transient stability studies or interarea oscillations [[1]](#1), allowing for fast simulations with bigger time steps than the EMT without loosing precision on these types of phenomena. 
+
+The assumptions made are:
+
+* The DC side of the converter is considered as an ideal DC voltage source $$E_{DC}$$.
+* The AC-side of the converter is connected to the grid through an AC filter formed by a resistance $$R$$ and an inductance $$L$$. It is represented as three current sources with magnitude and angle defined by the control system.
+* The system is considered to be balanced, and the positive and negative sequence are not considered.
+* The electrical variables are represented by phasors that rotate at the synchronous frequency of the grid in steady-state or quasi-steady-state.
+* The complete control structure from the EMT Model, apart from the PLL, is preserved. Further simplifications are made in the models that derive from the Full-Phasor model.
+
+The model has its limitations when performing high-frequency domain phenomena ($$>1$$ kHz), as all the fast-dynamics are not considered since it uses algebraic equations instead of the differential equations that are used on the [EMT model](../EMTGridFollowingVSC/). It is not useful either to calculate unbalanced situation as the model is based on the positive sequence.
+
+From this model, some approximations can be made to reduce the complexity of the model and speed up the simulation if the study to be performed does not require to cover some dynamics of the converter. These derived models can be consulted in Section [3.6](#derived-models)
```

Comment:
There is no section 3.6. Update it with the correct section.

Reply:
updated to section 4